examples/custom-resources/app-protect-waf-bundle/README.md

```diff
@@ -0,0 +1,72 @@
+# WAF
+
+In this example we deploy the NGINX Plus Ingress Controller with [NGINX App Protect](https://www.nginx.com/products/nginx-app-protect/), a web application and then configure load balancing and WAF protection for that application using the VirtualServer resource.
+
+## Prerequisites
+
+1. Follow the installation [instructions](https://docs.nginx.com/nginx-ingress-controller/installation) to deploy the Ingress Controller (including persisted volume PV) with NGINX App Protect. NGINX Ingress Controller deployment should include mounted volume (persisted volume PV). The WAF policy bundle file (```dataguard-alarm.tgz```) should be on the volume.
+1. Save the public IP address of the Ingress Controller into a shell variable:
+    ```
+    $ IC_IP=XXX.YYY.ZZZ.III
+    ```
+1. Save the HTTP port of the Ingress Controller into a shell variable:
+    ```
+    $ IC_HTTP_PORT=<port number>
+    ```
+
+## Step 1. Deploy a Web Application
+
+Create the application deployment and service:
+```
+$ kubectl apply -f webapp.yaml
+```
+
+## Step 2 - Configure the WAF Policy (Policy Bundle)
+
+Note: The policy bundle ```dataguard-alarm.tgz``` should already exist in the mounted volume in the NGINX Ingress Controller deployment.
```

Comment:
This should read like:
"Define an AppProtect Policy and retrieve its Bundle"
"Copy the bundle to the volume"